src/Infrastructure.VS/LanguageDetection/SonarLanguageRecognizer.cs

```diff
@@ -112,5 +119,34 @@ private static bool IsTypeScriptDocument(IEnumerable<IContentType> contentTypes)
         {
             return contentTypes.Any(type => type.IsOfType("TypeScript"));
         }
+
+        public AnalysisLanguage? GetAnalysisLanguageFromExtension(string fileName)
+        {
+            var extension = GetNormalizedExtention(fileName);
+
+            var ctype = fileExtensionRegistryService.GetContentTypeForExtension(extension);
```

Comment:
@ugras-ergun-sonarsource  SonarCloud is complaining that this variable is not used.

Reply:
I've already removed it and pushed.